packages/eslint-plugin-pf-codemods/src/rules/v6/chipReplaceWithLabel/chip-replace-with-label.test.ts

```diff
@@ -26,30 +26,30 @@ ruleTester.run('chip-replace-with-label', rule, {
       errors: [chipImportError],
     },
     {
-      code: `import { ChipGroup } from '@patternfly/react-core/deprecated'; <ChipGroup someOtherProp>This is a chipgroup</ChipGroup>`,
-      output: `import { LabelGroup } from '@patternfly/react-core'; <LabelGroup someOtherProp>This is a chipgroup</LabelGroup>`,
+      code: `import { ChipGroup } from '@patternfly/react-core/deprecated'; <ChipGroup numChips={3} someOtherProp>This is a chipgroup</ChipGroup>`,
```

Comment:
Rather than adding the old prop on several other existing tests, can we instead create a new test that tests specifically for the numChips prop? Similar to the test for isReadOnly prop

Reply:
i think the new tests that i added are valid test cases... so instead i reverted the more isolated tests to their originals, added an isolated numChips test, and kept the new tests as additional tests